skrf/network.py

```diff
@@ -4692,7 +4720,9 @@ def plot_attribute(     self,
                             y_label=None,
                             logx=False, **kwargs):
 
-
+        # create y_label if not provided
+        if y_label is None:
+            y_label = Network.Y_LABEL_DICT[conversion]
```

Comment:
@FranzForstmayr I am not sure it is the most proper way to make plots have y_label again. Have you eventually other ideas with the autogenerated plot functions? (cf. #1125)

Reply:
At least, ylabel are back!
![image](https://github.com/user-attachments/assets/48f87bde-3dd6-4dda-9ba7-a3f15896f382)